static/js/util/util.js

```diff
@@ -455,3 +455,30 @@ export function getUserDisplayName(profile: Profile): string {
 
   return `${first} ${last}${preferred_name}`;
 }
+
+function intersperse(arr, elementFunc) {
+  let resultArray = [], i=0;
+  if (i < arr.length)
+    resultArray.push(arr[i++]);
+  while (i < arr.length)
+    resultArray.push(elementFunc(i), arr[i++]);
+  return resultArray;
+}
+
+/**
+ * Takes a list of components and renders them with separator components in between each one
+ */
+export function renderSeparatedComponents(
+  components: Array<React$Element<*>|null>,
+  separator: string
+): Array<React$Element<*>|null> {
+  return intersperse(components, (i) => ( <span key={`separator-${i}`}>{ separator }</span> ));
+}
+
+/**
+ * Returns the value of an object's property by name, or a default if (a) the property name
+ * is null/undefined or (b) the property doesn't exist in the object.
+ */
+export function getPropertyOrDefault(obj: Object, property: ?string, defaultValue: any) {
+  return _.get(obj, property, defaultValue);
+}
```

Comment:
i created this for a couple reasons: (1) getPropertyOrDefault is a little bit clearer about intent than using _.get directly, and (2) there's a lot of functional overlap between ramda and lodash, so on the very very remote chance that we decided we wanted to get rid of lodash (or just import specific functions), we could change the implementation of this function. neither are terribly compelling reasons. i think it's likely for a developer to need to check docs whether they see this function or _.get with a third arg, and it's just a bit nicer to have that function documented locally, so i'm slightly leaning towards keeping it. let me know what you think